tests/UI/campaigns/functional/BO/13_shopParameters/02_orderSettings/02_statuses/returnStatuses/02_filterSortAndPagination.js

```diff
@@ -0,0 +1,301 @@
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByNameDesc', sortBy: 'name', columnID: 2, sortDirection: 'down',
+        },
+      },
+      {
+        args: {
+          testIdentifier: 'sortByIdAsc',
+          sortBy: 'id_order_return_state',
+          columnID: 1,
+          sortDirection: 'up',
+          isFloat: true,
+        },
+      },
+    ];
+
+    sortTests.forEach((test) => {
+      it(`should sort by '${test.args.sortBy}' '${test.args.sortDirection}' and check result`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', test.args.testIdentifier, baseContext);
+
+        let nonSortedTable = await statusesPage.getAllRowsColumnContent(
+          page,
+          tableName,
+          test.args.sortBy,
+          test.args.columnID,
+        );
+
+        await statusesPage.sortTable(page, tableName, test.args.sortBy, test.args.columnID, test.args.sortDirection);
+
+        let sortedTable = await statusesPage.getAllRowsColumnContent(
+          page,
+          tableName,
+          test.args.sortBy,
+          test.args.columnID,
+        );
+
+        if (test.args.isFloat) {
+          nonSortedTable = await nonSortedTable.map(text => parseFloat(text));
+          sortedTable = await sortedTable.map(text => parseFloat(text));
+        }
+
+        const expectedResult = await statusesPage.sortArray(nonSortedTable, test.args.isFloat);
+
+        if (test.args.sortDirection === 'up') {
+          await expect(sortedTable).to.deep.equal(expectedResult);
+        } else {
+          await expect(sortedTable).to.deep.equal(expectedResult.reverse());
+        }
+      });
+    });
+  });
+
+  // 3 - Create 16 order return statuses
+  const creationTests = new Array(16).fill(0, 0, 16);
+
+  creationTests.forEach((test, index) => {
+    describe(`Create order return status n°${index + 1} in BO`, async () => {
+      const orderReturnStatusData = new OrderReturnStatusFaker({name: `todelete${index}`});
+
+      it('should go to add new order status group page', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `goToAddOrderReturnStatusPage${index}`, baseContext);
+
+        await statusesPage.goToNewOrderReturnStatusPage(page);
+
+        const pageTitle = await addOrderReturnStatusPage.getPageTitle(page);
+        await expect(pageTitle).to.contains(addOrderReturnStatusPage.pageTitleCreate);
+      });
+
+      it('should create order status and check result', async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `createOrderReturnStatus${index}`, baseContext);
+
+        await addOrderReturnStatusPage.setOrderReturnStatus(page, orderReturnStatusData);
+        /* Successful message is not visible, skipping it */
+        /* https://github.com/PrestaShop/PrestaShop/issues/21270 */
+        // await expect(textResult).to.contains(statusesPage.successfulCreationMessage);
```

Comment:
Do we have a list of skipped steps or actions like this ?
Could be done in skipped test file. (to keep track and change it when it's fixed)